Makefile

```diff
@@ -415,3 +415,10 @@ show-container-engine:
 	@echo -n $(CONTAINER_ENGINE)
 
 
+imports: ## Organize imports in go files using goio. Example: make imports
+	go run ./vendor/github.com/go-imports-organizer/goio
```

Comment:
I thought we wanted to postpone the decision of vendoring, no? Otherwise, that's a lot of files - makes it even impossible to load the PR files view from my browser.. :-)

Since `go-imports-organizer` can be installed as a standalone binary, can we do a similar thing as what we are doing to download other tools like ginkgo, controller-gen or envtest, i.e., download the binaries to the local `bin` folder if they are missing and use them from there? See https://github.com/janus-idp/operator/blob/main/Makefile#L254-L262

Reply:
I can rework the pr to remove the vendoring, though i am pretty sure we will need to add it at some point.

Reply:
@coreydaley yes, let's decouple this issue and vendoring please and go with this tool as we do with other (as Armel suggested).

Reply:
Done.